src/db.rs

```diff
@@ -580,6 +580,10 @@ impl Db {
         fp_registry: Arc<FailPointRegistry>,
     ) -> Result<Self, SlateDBError> {
         let path = path.into();
+        tracing::info!("Opening database at path: {:?}", path);
+        if let Ok(options_json) = options.to_json_string() {
+            tracing::warn!("Using options: {}", options_json);
+        }
```

Comment:
we might want to default to logging using `{:?}` if we can't serialize it into JSON